README.md

```diff
@@ -37,7 +37,7 @@ To learn more about Cortex, consult the following talks and articles.
 
 ### Recent talks and articles
 
-- Dec 2020 blog post "[How AWS and Grafana Labs are scaling Cortex for the cloud](https://aws.amazon.com/blogs/opensource/how-aws-and-grafana-labs-are-scaling-cortex-for-the-cloud/)
+- Dec 2020 blog post "[How AWS and Grafana Labs are scaling Cortex for the cloud](https://aws.amazon.com/blogs/opensource/how-aws-and-grafana-labs-are-scaling-cortex-for-the-cloud/)"
```

Comment:
Could you also fix `docs/_index.md` please?

Reply:
Done